python-backend/uwsgi.ini

```diff
@@ -0,0 +1,31 @@
+[uwsgi]
+# Module settings
+callable = create_app()
+module = app
+
+# Curl this address for statistics
+stats = 127.0.0.1:9191
+
+# Server settings
+http-socket = 0.0.0.0:5000
+http-auto-gzip = true
+post-buffering = 8192
+
+# UWSGI settings
+plugins = router_cache
+master = true
+harakiri = 90
+processes = 4
+threads = 2
+enable-threads = true
+vacuum = true
+disable-logging = True
+
+# Cache
+cache2 = name=mines,items=100,keysize=10240
+route-if = contains:${HTTP_ACCEPT_ENCODING};gzip cache:key=gzipped_${REQUEST_URI},name=mines,content_encoding=gzip,expires=3600
+route = ^/$ cache:key=${REQUEST_URI},name=mines,expires=3600
+route = ^/$ cachestore:key=${REQUEST_URI},gzip=gzipped_mines_homepage,name=mines,expires=3600
```

Comment:
Do you understand the cachestore? Does this mean a newly added mine would not appear on the endpoint for 3600 seconds?

Reply:
Not really. I don't intend to ship it off with this PR. Just wanted to play around and see how it behaves.

I dont think cache should be automagically controlled by a library coz we can run into magic issues. I just wanted to test and see how it behaves.

I'll play around with that a little tonight but if it's not working great, i'll remove it and we .can revisit that in future.